### ISSUE TITLE
Duplicate Dashboards

### ISSUE THREAD

**Issue by swiffer** [teslamate-org/teslamate#4298]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

We currently do have multiple dashboards showing similar graphs - especially:

- Mileage
- Projected Range
- Charge Level

While I do see additional value in Charge Level Dashboard with the additional data that has been added recently the Mileage Dashboard seems obsolete with having a Projected Range Dashboard that has Mileage in it's first graph.

### Relevant log output

```shell
-
```

### Type of installation

Docker

### Version

v1.30.1

**Comment by JakobLichterfeld** [comment]
Screenshots: https://docs.teslamate.org/docs/screenshots/

**Comment by JakobLichterfeld** [comment]
> the Mileage Dashboard seems obsolete with having a Projected Range Dashboard that has Mileage in it's first graph.

I tend to agree. What do you think @DrMichael, should we remove Mileage Dashboard?

**Comment by DrMichael** [comment]
> I tend to agree. What do you think @DrMichael, should we remove Mileage Dashboard?

Yes, fine with me. One could also think about the States dashboard...

**Comment by JakobLichterfeld** [comment]
> One could also think about the States dashboard...

Yes and no, imo this is something to quickly check just the states, otherwise harder to find which dashboard does have the states, if you do not know overview or on mobile where overview is not ideal for checking states.

**Comment by DrMichael** [comment]
Ah, mobile is a good point...

**Comment by swiffer** [comment]
If anyone comes up with a good name under which those 3 Dashboards could be combined I'll do it

**Comment by JakobLichterfeld** [comment]
Mich überzeugt keiner: ![Image](https://github.com/user-attachments/assets/29504e0b-d595-45ac-b751-8efb73d5f676)

**Comment by JakobLichterfeld** [comment]
Etwas besser:

![Image](https://github.com/user-attachments/assets/fc957ed5-98a8-44cc-8beb-0578aa3be59b)
